### PR TITLE
Feature scope

### PR DESCRIPTION
**Pull request by vlal**
* Add scope to k8s dashboard

As per #1577

### CODE REVIEW

File: src/app/backend/resource/scope/scope.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Workloads structure contains all resource lists grouped into the workloads category.
+type Scope struct {
+	Deployed bool   `json:"deployed"`
+	Address  string `json:"address"`
+}
+
+// GetScope returns a populated Scope struct.
```
Comment (olekzabl) [inline src/app/backend/resource/scope/scope.go]: Copy-pasted comment?

File: src/app/frontend/scope/controller.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      });
+    });
+  }
+
+  /**
+   * Pings until scope is up and ready.
+   * @export
+   */
+  pingScope(tries) {
+    if (tries < 5) {
```
Comment (olekzabl) [inline src/app/frontend/scope/controller.js]: Calling pingScope(5) will do nothing, right? I initially was a little confused by this.
Can you refactor this argument (e.g. to decrease until 0), or add a comment about its meaning?

File: src/app/backend/resource/scope/scope.go [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+type Scope struct {
+	Deployed bool   `json:"deployed"`
+	Address  string `json:"address"`
+}
+
+// PostScope deploys Scope onto the cluster.
+func PostScope(client client.Interface) (*Scope, error) {
+	log.Print("Installing scope on the cluster")
+
+	response, err := http.Get("https://cloud.weave.works/launch/k8s/weavescope.yaml?k8s-service-type=NodePort")
```
Comment (bryk) [inline src/app/backend/resource/scope/scope.go]: This is the part I'm worried about. It allows cloud.weave.works to hack user's cluster. Can we embed the YAML in repository here and versionize it? It'd be a bit safer at least.

Comment (errordeveloper) [reply src/app/backend/resource/scope/scope.go]: @bryk I believe that's been resolved now. Could you please review the rest of this PR and let @vlal and myself know if you broadly approve of it before we rebase and update? Thanks!

**Comment by errordeveloper** [comment]
Let's get back to this!

/remove-lifecycle stale

cc @danielromlein @bryk

**Comment by maciaszczykm** [comment]
@errordeveloper I think it is not the best time. We are migrating the whole app to new Angular and there is no sense in implementing it for AngularJS. Let's wait for #2727 and return to this discussion then.

**Comment by errordeveloper** [comment]
Thanks, Marcin! We shall wait.

**Pull request closed without merging**